### PR TITLE
fix(nms): nginx ssl issue

### PR DESCRIPTION
**Pull request by prabinakpattnaik**
Below error is observed when creating orc8r pods:
2023/06/15 13:34:36 [emerg] 1#1: unknown directive "ssl" in /etc/nginx/conf.d/nginx_proxy_ssl.conf:3
nginx: [emerg] unknown directive "ssl" in /etc/nginx/conf.d/nginx_proxy_ssl.conf:3

orc8r            nms-nginx-proxy-7454448447-xzc8l                 0/1     CrashLoopBackOff   7 (119s ago)   16m

### CODE REVIEW

**Review by maxhbr — APPROVED** [review]
Code changes look good.

**Comment by maxhbr** [comment]
CI is red, due to out of disk space in a build:




and nms yarn test is red: https://github.com/magma/magma/actions/runs/5280377687/jobs/9558018580?pr=15215

**Comment by prabinakpattnaik** [comment]
@maxhbr , any idea why "nms yarn test" is failed. Is it lint error or test case failed error.

**Comment by maxhbr** [comment]
you can look in the log and see the test summary:

```
Summary of all failing tests
FAIL app/views/traffic/__tests__/TrafficOverviewTest.tsx (8.891 s)
  ● <TrafficDashboard APNs/> › renders

    TypeError: Cannot redefine property: location

      319 |
      320 |   beforeEach((): void => {
    > 321 |     window.location = {
          |     ^
      322 |       pathname: '/nms/test/traffic/apn',
      323 |     } as Location;
      324 |

      at Object.<anonymous> (app/views/traffic/__tests__/TrafficOverviewTest.tsx:321:5)

  ● <TrafficDashboard APNs/> › renders

    TypeError: Cannot redefine property: location

      340 |
      341 |   afterEach((): void => {
    > 342 |     window.location = location;
          |     ^
      343 |   });
      344 |
      345 |   const Wrapper = () => (

      at Object.<anonymous> (app/views/traffic/__tests__/TrafficOverviewTest.tsx:342:5)

  ● <TrafficDashboard APNs/> › shows prompt when remove apn is clicked

    TypeError: Cannot redefine property: location

      319 |
      320 |   beforeEach((): void => {
    > 321 |     window.location = {
          |     ^
      322 |       pathname: '/nms/test/traffic/apn',
      323 |     } as Location;
      324 |

      at Object.<anonymous> (app/views/traffic/__tests__/TrafficOverviewTest.tsx:321:5)

  ● <TrafficDashboard APNs/> › shows prompt when remove apn is clicked

    TypeError: Cannot redefine property: location

      340 |
      341 |   afterEach((): void => {
    > 342 |     window.location = location;
          |     ^
      343 |   });
      344 |
      345 |   const Wrapper = () => (

      at Object.<anonymous> (app/views/traffic/__tests__/TrafficOverviewTest.tsx:342:5)


Test Suites: 1 failed, 4 skipped, 52 passed, 53 of 57 total
Tests:       2 failed, 4 skipped, 349 passed, 355 total
```

there are test failures

**Comment by prabinakpattnaik** [comment]
Yes, I have seen yesterday on that logs but could not get the clue for whether these errors are link with changes or existing error.

**Comment by maxhbr** [comment]
it seems to be a general issue that needs to be resolved. Also other PRs have the same issues in the log https://github.com/magma/magma/actions/runs/5247558970/jobs/9477884882?pr=15211

**Comment by prabinakpattnaik** [comment]
@maxhbr , can we do force merge the changes as both CI checks are failing because of one is know issue and another is disk space issue.

**Comment by maxhbr** [comment]
see https://magmacore.slack.com/archives/C01Q1T14YJ2/p1686916038724709?thread_ts=1686905954.161159&cid=C01Q1T14YJ2, since there is no process for force merges.

File: nms/app/views/traffic/__tests__/TrafficOverviewTest.tsx
Comment (maxhbr) [inline nms/app/views/traffic/__tests__/TrafficOverviewTest.tsx]: if you start working on fixing CI runs, force merges are no longer an option. Would also fit to be put into its own PR?

**Comment by lucasgonze** [comment]
Seems like the underlying failure is meaningful: 

>   Warning: A component is changing a controlled input to be uncontrolled. This is likely caused by the value changing from a defined to undefined, which should not happen. Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://reactjs.org/link/controlled-components

However, the code in this commit doesn't do anything in NMS, so the problem must be a prior commit. Do you agree?

**Comment by maxhbr** [comment]
see: https://github.com/magma/magma/pull/15215#issuecomment-1594401817

this error already appeared in other CI runs before this change

File: nms/app/views/traffic/__tests__/TrafficOverviewTest.tsx
Comment (maxhbr) [reply nms/app/views/traffic/__tests__/TrafficOverviewTest.tsx]: Hey, can you remove that commit again, so that the impact of a potential force merge is smaller? Also: the commit message is not very helpful.

Comment (panyogesh) [reply nms/app/views/traffic/__tests__/TrafficOverviewTest.tsx]: Sure.

Comment (maxhbr) [reply nms/app/views/traffic/__tests__/TrafficOverviewTest.tsx]: ping: can someone drop that commit, so that we can move forward?

**Comment by maxhbr** [comment]
Good, that change can be discussed there and the decision whether it is appropriate does not block this PR.

**Comment by maxhbr** [comment]
@prabinakpattnaik : you still need to drop https://github.com/magma/magma/pull/15215/commits/34da9f84de992461dd2370639abb43a2287a90dc from this PR to unblock it.

**Comment by prabinakpattnaik** [comment]
@maxhbr , I am doing now.

**Comment by maxhbr** [comment]
now it does the change, and reverts it again. But after squashing that should be hidden.

Now we just need to find someone that is able to force-merge. This was made harder with some of the recent policy changes (ping @lucasgonze). see also: https://magmacore.slack.com/archives/C01Q1T14YJ2/p1687165644395929?thread_ts=1686905954.161159&cid=C01Q1T14YJ2

**Pull request merged**
